packages/terra-application-menu-layout/src/ApplicationMenuLayout.scss

```diff
@@ -0,0 +1,48 @@
+/* stylelint-disable selector-max-compound-selectors */
+/* stylelint-disable max-nesting-depth */
+:local {
+  .fit {
+    flex: 0 0 auto;
+  }
+
+  .fill {
+    flex: 1 1 auto;
+    flex-direction: column;
+    height: 100%;
+    justify-content: space-between;
+  }
+
+  .fit,
+  .fill {
+    display: flex;
+    position: relative;
+    width: 100%;
+  }
+
+  .menu {
+    > .header,
+    > .footer {
+      max-height: 40%;
+    }
+
+    > .body {
+      > .widgets {
+        max-height: 100%;
+      }
+
+      > .content {
+        overflow: hidden;
+
+        > .normalizer {
```

Comment:
What does normalizer mean in this context? I find it to be a confusing classname.

Reply:
This is matching a similar pattern to what we have in the content container. A div that, through the use of absolute positioning, prevents flexbox styles from causing layout issues for the content provided to the menu. I'm up for suggestions if we want to name it something else.